### PR TITLE
Bring Facebook\Exceptions and tests into PSR-2 compliance

### PR DESCRIPTION
**Pull request by rpalladino**

### CODE REVIEW

File: src/Facebook/Exceptions/FacebookAuthenticationException.php [hunk 1]
```diff
@@ -30,4 +30,4 @@
 class FacebookAuthenticationException extends FacebookSDKException
 {
```
Comment (yguedidi) [inline src/Facebook/Exceptions/FacebookAuthenticationException.php]: All empty line in empty classes should be removed

File: src/Facebook/Exceptions/FacebookResponseException.php [hunk 2]
```diff
@@ -32,182 +32,181 @@
...
+     * @return FacebookResponseException
+     */
+    public static function create(FacebookResponse $response)
+    {
+        $data = $response->getDecodedBody();
+
+        if (! isset($data['error']['code']) && isset($data['code'])) {
+            $data = ['error' => $data];
+        }
+        $code = isset($data['error']['code']) ? $data['error']['code'] : null;
```
Comment (yguedidi) [inline src/Facebook/Exceptions/FacebookResponseException.php]: Empty line above

File: src/Facebook/Exceptions/FacebookResponseException.php [hunk 3]
```diff
@@ -32,182 +32,181 @@
...
+            switch ($data['error']['error_subcode']) {
+                // Other authentication issues
+                case 458:
+                case 459:
+                case 460:
+                case 463:
+                case 464:
+                case 467:
+                    return new static($response, new FacebookAuthenticationException($message, $code));
+                    break;
```
Comment (yguedidi) [inline src/Facebook/Exceptions/FacebookResponseException.php]: Should be removed

Comment (yguedidi) [inline src/Facebook/Exceptions/FacebookResponseException.php]: And all below `break`s that follow a `return`

File: src/Facebook/Exceptions/FacebookResponseException.php [hunk 4]
```diff
@@ -32,182 +32,181 @@
...
+     * @param mixed $default
+     *
+     * @return mixed
+     */
+    private function get($key, $default = null)
+    {
+        if (isset($this->responseData['error'][$key])) {
+            return $this->responseData['error'][$key];
+        }
+        return $default;
```
Comment (yguedidi) [inline src/Facebook/Exceptions/FacebookResponseException.php]: Empty line above

**Comment by yguedidi** [comment]
Please see #356

**Comment by gfosco** [comment]
Closing due to #356 and travis issues.  If this is still valid after #356 please resubmit then.

**Pull request closed without merging**